web/app/js/components/util/TapUtils.jsx

```diff
@@ -135,34 +135,66 @@ const publicAddressToString = ipv4 => {
 /*
   display more human-readable information about source/destination
 */
-export const srcDstColumn = (display, labels, ResourceLink) => {
-  let podLink = (
-    <ResourceLink
-      resource={{ type: "pod", name: labels.pod, namespace: labels.namespace }}
-      linkText={"po/" + labels.pod} />
+const podLink = (labels, ResourceLink) => (
+  <ResourceLink
+    resource={{ type: "pod", name: labels.pod, namespace: labels.namespace }}
+    linkText={"po/" + labels.pod} />
+);
+
+const deployLink = (labels, ResourceLink) => (
+  <ResourceLink
+    resource={{ type: "deployment", name: labels.deployment, namespace: labels.namespace}}
+    linkText={"deploy/" + labels.deployment} />
+);
```

Comment:
Related to my other comment below -- I'd prefer to generalize these methods to work for all resource types, not just deployments and pods. I think something like this would work:

```
const resourceLink = (resourceType, labels, ResourceLink) => (
  <ResourceLink
    resource={{ type: resourceType, name: labels[resourceType], namespace: labels.namespace}}
    linkText={toShortResourceName(resourceType) + "/" + labels[resourceType]} />
);
```